src/pages/profiles/ProfileList.tsx

```diff
@@ -56,62 +129,122 @@ const ProfileList: FC = () => {
           "aria-label": "Name",
         },
         {
-          content: profile.description,
+          content: (
+            <div className="u-truncate" title={profile.description}>
+              {profile.description}
+            </div>
+          ),
           role: "rowheader",
           "aria-label": "Description",
         },
         {
-          content: getProfileInstances(
-            project,
-            isDefaultProject,
-            profile.used_by
-          ).length,
+          content: (
+            <>
+              {isDefaultProject && (
+                <>
+                  {total} in all projects
```

Comment:
Can we hide this if the usedBy matches the total?

Reply:
Maybe it should instead say "x instances in other projects" on a 2nd line?

Reply:
This is a question for @al-bakalova

Reply:
I am not sure I get the question. I'd prefer to always keep both lines and never hide (for consistency reasons).

10 instances
10 in all projects

It kind of currently repeats the total but gives the user an idea that the profiles of the default project are actually used across different projects.